eve/main.yml

```diff
@@ -68,4 +68,4 @@ stages:
     - Git: *git_pull
     - ShellCommand:
         name: 'Build doc'
-        command: tox -e docs
+        command: tox --workdir /tmp/tox -e docs -- html latexpdf
```

Comment:
Wouldn't it be also interesting to add an `Upload` step so that you can retrieve the pdf/html documentation in case you want to see the rendering?

Reply:
Over time, maybe, however, right now docs are rendered and served on ReadTheDocs already. Figuring out the whole docs workflow is under discussion, also related to our other projects.